content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```diff
@@ -31,7 +31,33 @@ $ java -jar GameLiftLocal.jar -p 9080
 
 ```
 
-## 2. Start Server
+## 2. Set `sv_useGameLiftServer` to `true`
+
+When you are ready to test with GameLift, opt-in by setting the `sv_useGameLiftServer` CVAR to `true`. This CVAR is off by default to facilitate local testing prior to the integration of the GameLift Server SDK into your game server.
+
+You can set this value by creating a settings registry file named **commands.MYPROJECT_serverlauncher.setreg** in the **YourProject/Registry** folder with the following contents:
```

Comment:
- "YourProject" --> "MyProject", so it's clear we're talking about the same project. 
I also wrote them the both in camel case, please verify if this is correct. Should the project name in the `.setreg` file be all upper case?
- Use `code-style` formatting for files and paths

```suggestion
You can set this value by creating a settings registry file named `commands.MyProject_serverlauncher.setreg` in the `MyProject/Registry` folder with the following contents:
```